src/plugins/console/public/lib/mappings/mappings.js

```diff
@@ -54,20 +51,23 @@ export function expandAliases(indicesOrAliases) {
   if (typeof indicesOrAliases === 'string') {
     indicesOrAliases = [indicesOrAliases];
   }
-  indicesOrAliases = $.map(indicesOrAliases, function (iOrA) {
+
+  indicesOrAliases = indicesOrAliases.map((iOrA) => {
     if (perAliasIndexes[iOrA]) {
       return perAliasIndexes[iOrA];
     }
     return [iOrA];
   });
   let ret = [].concat.apply([], indicesOrAliases);
   ret.sort();
-  let last;
-  ret = $.map(ret, function (v) {
-    const r = last === v ? null : v;
-    last = v;
-    return r;
-  });
+  ret = ret.reduce((result, value, index, array) => {
+    const last = array[index - 1];
+    if (last !== value) {
+      result.push(value);
+    }
+    return result;
```

Comment:
nit - this is fine, but all we're really trying to do is get an array with the unique values, right? Something like

```
const ret = [...new Set(indicesOrAliases)]]
```